### DISCUSSION TITLE
Slides Per View

### DISCUSSION THREAD

**Discussion by paulsizer** [nolimits4web/swiper#4194] [Q&A] [answered]
I have created a swiper container with a SlidesPerView set to 2, if I have only one slide this is left aligned in the container. Is there a way to set this to be right aligned. There is an option for centered slides, and it defaults to left but no option for right aligned.

**Comment by nolimits4web** [comment, marked as answer]
Codepen/JSFiddle/Codesandbox and others live example can really help to answer your question

**Reply by paulsizer** [reply]
I was lazy in asking my question I apologise! When I should have tried to find a solution and not a quick fix.

I solved my issue using the documentation and using the events that where already there. 

If it helps anyone else this is the approach I used:

`
const swiper = new Swiper('.swiper-container', {
  on: {
    init: function (e) {
          if (e.slides.length === 1) {
            e.slides[0].parentElement.classList.add("single-slide");
          }
      },
  }
});
`
